### PR TITLE
ROX-17546: Remove support for Central PVC from Helm Chart

### PR DESCRIPTION
**Pull request by c-du**
### Description
This is the first part of deprecate the central support for "stackrox-db" PVC.

### User-facing documentation

- [ ] CHANGELOG is updated **OR** update is not needed
- [ ] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request) is created and is linked above **OR** is not needed

### Testing

- [ ] inspected CI results

#### Automated testing

- [ ] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [ ] modified existing tests

#### How I validated my change

change me!

### CODE REVIEW

File: tests/upgrade/lib.sh [hunk 1]
```diff
@@ -99,7 +99,6 @@ deploy_earlier_postgres_central() {
     helm install -n stackrox --create-namespace stackrox-central-services /tmp/early-stackrox-central-services-chart \
          --set central.adminPassword.value="${ROX_PASSWORD}" \
          --set central.db.enabled=true \
-         --set central.persistence.none=true \
```
Comment (dashrews78) [inline tests/upgrade/lib.sh:102]: Might need this one.  It is deploying an earlier release, not the current one.

File: image/templates/helm/stackrox-central/templates/01-central-02-psps.yaml [hunk 2]
```diff
@@ -56,10 +56,6 @@ spec:
     - 'downwardAPI'
     - 'persistentVolumeClaim'
     - 'hostPath'
```
Comment (dashrews78) [inline image/templates/helm/stackrox-central/templates/01-central-02-psps.yaml:58]: Do we still need `hostPath` here?

File: tests/e2e/run-scanner-v4.bats [hunk 3]
```diff
@@ -225,7 +225,14 @@ teardown_file() {
         MAIN_IMAGE_TAG=$EARLIER_MAIN_IMAGE_TAG
         info "Overriding MAIN_IMAGE_TAG=$EARLIER_MAIN_IMAGE_TAG"
     fi
-    CENTRAL_CHART_DIR_OVERRIDE="${_CENTRAL_CHART_DIR_OVERRIDE}" _deploy_stackrox
+    local _ROX_CENTRAL_EXTRA_HELM_VALUES_FILE
+    _ROX_CENTRAL_EXTRA_HELM_VALUES_FILE=$(mktemp)
+    cat <<EOF >"$_ROX_CENTRAL_EXTRA_HELM_VALUES_FILE"
+central:
+  persistence:
+    none: true
```
Comment (dashrews78) [inline tests/e2e/run-scanner-v4.bats:233]: what is this for?

File: image/templates/helm/stackrox-central/templates/01-central-02-psps.yaml (hunk 2 above)
Comment (c-du) [reply image/templates/helm/stackrox-central/templates/01-central-02-psps.yaml]: Removed.

**Review by dashrews78 — APPROVED** [review]

**Pull request merged**